frontend/Dockerfile.e2e

```diff
@@ -0,0 +1,20 @@
+FROM node:12 AS production
```

Comment:
I think the `AS production` is unnecessary here?

Reply:
Good point